definition/data/sheets/CaseEventToComplexTypes.json

```diff
@@ -5,5 +5,23 @@
-    {"ID": "ScannedDocument","CaseEventID": "ReviewedPaperResponse","CaseFieldID": "scannedDocuments","ListElementCode": "subtype","EventElementLabel": "Document Subtype","EventHintText": "","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"}
-]
\ No newline at end of file
+    {"ID": "ScannedDocument","CaseEventID": "ReviewedPaperResponse","CaseFieldID": "scannedDocuments","ListElementCode": "subtype","EventElementLabel": "Document Subtype","EventHintText": "","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"},
+    
+    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "extraOrderDirection","CaseFieldID": "otherDirections","EventElementLabel": "Select direction","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "OPTIONAL"},
+    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "extraDocUploadList","CaseFieldID": "otherDirections","EventElementLabel": "Extra statements for document upload","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "MANDATORY"},
+    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "expertReports","CaseFieldID": "otherDirections","EventElementLabel": "Extra statements for expert report","EventHintText": "","FieldDisplayOrder": 1,"DisplayContext": "OPTIONAL"},
+    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "otherDirectionHeaders","CaseFieldID": "otherDirections","EventElementLabel": "Direction header","EventHintText": "e.g. Upload contracts /Confirm documents are original","FieldDisplayOrder": 2,"DisplayContext": "MANDATORY"},
+    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "directionComment","CaseFieldID": "otherDirections","EventElementLabel": "Direction comment","EventHintText": "","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"},
+    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "sendBy","CaseFieldID": "otherDirections","EventElementLabel": "Response deadline","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "OPTIONAL"},
+    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "forParty","CaseFieldID": "otherDirections","EventElementLabel": "Direction is for","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "MANDATORY"},
+    
+    {"ID": "OtherDirection","CaseEventID": "ActionReviewComments","ListElementCode": "extraOrderDirection","CaseFieldID": "otherDirections","EventElementLabel": "Select direction","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "OPTIONAL"},
+    {"ID": "OtherDirection","CaseEventID": "ActionReviewComments","ListElementCode": "extraDocUploadList","CaseFieldID": "otherDirections","EventElementLabel": "Extra statements for document upload","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "MANDATORY"},
```

Comment:
Suggestions - Instead of Extra , can it be Additional ?

Reply:
id or label? will change in another PR

Reply:
few done here